roles/ci_gen_kustomize_values/tasks/generate_snippets.yml

```diff
@@ -110,4 +104,5 @@
     - name: Expose the datatype for future usage
       ansible.builtin.set_fact:
         values_datatype: "{{ _datatype }}"
+        snippet_datadir: "{{ _dest_dir }}"
```

Comment:
Small "cosmetic" point. If the block in [line 17](https://github.com/openstack-k8s-operators/ci-framework/pull/1166/files#diff-abf0c432b7fa9073946612b0011782cd9595b59432cba193f66fde8aab7b6513R17) is here only to expose this two variable to all the tasks in the block, but, those variables need to be consumed outside the file. Why not just move this fact up before the previous referenced line and get rid of the entire block? That way, apart of getting rid of one level of indentation we would use one two facts to hold those same values instead of four.

Reply:
I need to double-check if those facts are really consumed outside.

Reply:
'k, done, though it's not really improving anything at this point. More cosmetics.